guides/hack/21-XHP/25-further-resources.md

```diff
@@ -1,5 +1,3 @@
 Besides our documentation, there are some other very useful resources to enhance your experience with XHP.
 * [XHP Library](https://github.com/facebook/xhp-lib): The class libraries for XHP.
-* [XHP Bootstrap Library](http://github.com/hhvm/xhp-bootstrap): A framework providing XHP components for web pages.
-* [Using XHP with Bootstrap](http://hhvm.com/blog/6515/using-xhp-with-bootstrap): A blog post about using XHP with the bootstrap library.
```

Comment:
If you removed this link because it 404'ed, please replace it with https://github.com/facebookarchive/xhp-bootstrap/ instead. From the project status:

We are not planning to actively maintain this project; however, we still believe it has value as an example.

Reply:
I feel these should be entirely removed - as well as being unmaintained:
- they're now considered a bad idea
- they target v3

Reply:
@lexidor - as already said, Fred and I had a conversation offline and are opting to fully remove.

Reply:
If we remove links to xhp-bootstrap, I should probably remove this one too.

https://github.com/hhvm/xhp-lib/commit/453325f48cf8be799d7332e0e90cbec127b88c4e#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R16